packages/shared/src/utils/constants/contract.tsx

```diff
@@ -399,17 +411,17 @@ export const getSupportedContracts = is_high_low => ({
     },
 });
 
-export const getContractConfig = is_high_low => ({
+export const getContractConfig = (is_high_low: boolean) => ({
     ...getSupportedContracts(is_high_low),
     ...getUnsupportedContracts(),
 });
 
-export const getContractTypeDisplay = (type, is_high_low = false) => {
-    return getContractConfig(is_high_low)[type] ? getContractConfig(is_high_low)[type.toUpperCase()].name : '';
+export const getContractTypeDisplay = (type: TGetSupportedContracts, is_high_low = false) => {
+    return getContractConfig(is_high_low)[type].name || '';
```

Comment:
Two suggestions:

1. `return` is not needed here.
2. Can't we merge `getContractTypeDisplay` and `getContractTypePosition` functions? the difference between these two functions is just the property they return. (name/position)

Reply:
@hamid-deriv Thank u for your revision, since these changes are out of the scope of this card, I think it's better to fix them later.
We have many files changes in the parent PR which is really big.

Reply:
:+1: 

I think it would be good if we place a TODO or note these changes to remember to change them later.

Reply:
sure, will add it.